util/util.go

```diff
@@ -268,3 +269,19 @@ func RemoveKeyPrefix(key string) (string, error) {
 	}
 	return match[3], nil
 }
+
+// PrintMemUsage outputs the current, total and OS memory being used. As well as the number
+// of garage collection cycles completed.
```

Comment:
We could also consider using https://cloud.google.com/profiler/docs/profiling-go

Reply:
Good point, added to https://github.com/datacommonsorg/mixer/issues/347 and can do for the next PR